common/list/lists-mws-en.html

```diff
@@ -0,0 +1,241 @@
+</ol>
+<h3>Code</h3>
+<pre><code>&lt;ol&gt;
+	&lt;li&gt;...
+		&lt;div <strong>class=&quot;lst-num&quot;</strong>&gt;
+			&lt;ol&gt;
+				&lt;li&gt;...&lt;/li&gt;
+			&lt;/ol&gt;
+		&lt;/div&gt;
+	&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+
+<h2>Alpha lists − Lower alpha</h2>
+<ol>
+	<li>List item 1</li>
+	<li>
+		List item 2
+		<div class="lst-lwr-alph">
+			<ol>
+				<li>List item 2a</li>
+				<li>List item 2b</li>
+			</ol>
+		</div>
+	</li>
+	<li>List item 3</li>
+</ol>
+<h3>Code</h3>
+<pre><code>&lt;ol&gt;
+	&lt;li&gt;...
+		&lt;div <strong>class=&quot;lst-lwr-alph&quot;</strong>&gt;
+			&lt;ol&gt;
+				&lt;li&gt;...&lt;/li&gt;
+			&lt;/ol&gt;
+		&lt;/div&gt;
+	&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+
+<h2>Alpha lists − Upper alpha</h2>
+<div class="lst-upr-alph">
+	<ol>
+		<li>List item 1</li>
+		<li>
+			List item 2
+			<ol>
+				<li>List item 2a</li>
+				<li>List item 2b</li>
+			</ol>
+		</li>
+		<li>List item 3</li>
+	</ol>
+</div>
+<h3>Code</h3>
+<pre><code>&lt;div <strong>class=&quot;lst-upr-alph&quot;</strong>&gt;
+	&lt;ol&gt;
+		&lt;li&gt;...
+			&lt;ol&gt;
+				&lt;li&gt;...&lt;/li&gt;
+			&lt;/ol&gt;
+		&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;div&gt;</code></pre>
+
+<h2>Roman numeral − Lower Roman</h2>
+<ol>
+	<li>List item 1</li>
+	<li>
+		List item 2
+		<div class="lst-lwr-rmn">
+			<ol>
+				<li>List item 2a</li>
+				<li>List item 2b</li>
+			</ol>
+		</div>
+	</li>
+	<li>List item 3</li>
+</ol>
+<h3>Code</h3>
+<pre><code>&lt;ol&gt;
+	&lt;li&gt;...
+		&lt;div <strong>class=&quot;lst-lwr-rmn&quot;</strong>&gt;
+			&lt;ol&gt;
+			&lt;li&gt;...&lt;/li&gt;
+			&lt;/ol&gt;
+		&lt;/div&gt;
+	&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+
+
+<h2>Roman numeral − Upper Roman</h2>
+<div class="lst-upr-rmn">
+<ol>
+	<li>List item 1</li>
+	<li>
+		List item 2
+		<ol>
+			<li>List item 2a</li>
+			<li>List item 2b</li>
+		</ol>
+	</li>
+	<li>List item 3</li>
+</ol>
+</div>
+<h3>Code</h3>
+<pre><code>&lt;div<strong> class=&quot;lst-upr-rmn&quot;</strong>&gt;
+	&lt;ol&gt;
+		&lt;li&gt;...&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;/div&gt;</code></pre>
+
+<h2>List inline</h2>
+<div class="list-inline">
+<ul>
+	<li>List item 1</li>
+	<li>List item 2</li>
+	<li>List item 3</li>
+</ul>
+</div>
+<h3>Code</h3>
+<pre><code>&lt;div <strong>class=&quot;list-inline&quot;</strong>&gt;
+	&lt;ol&gt;
+		&lt;li&gt;...&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;/div&gt;</code></pre>
+
+<h2>Spacing lists - spaces added</h2>
+<div class="lst-spcd">
+	<ul>
+		<li>List item 1</li>
+		<li>List item 2</li>
+		<li>List item 3</li>
+	</ul>
+</div>
+<h3>Code</h3>
+<pre><code>// Spaced list:
+&lt;div <strong>class=&quot;lst-spcd&quot;</strong>&gt;
+	&lt;ul&gt;
+		&lt;li&gt;List item 1&lt;/li&gt;
+		&lt;li&gt;List item 2&lt;/li&gt;
+		&lt;li&gt;List item 3&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;</code></pre>
+
+<h2>List - Additional styles - GCWeb theme V5</h2>
+<h3>List double spaced <code>.lst-spcd-2</code> </h3>
```

Comment:
```suggestion
<h2>List double spaced <code>.lst-spcd-2</code></h2>
```

Un titre devrait contenir plusieurs sous-section ou aucun. Du mois il devrait apporté un avantage lié à l'organisation du contenu pour le public cible. Ici c'est destiné à informé les web author.

Il n'est pas nécessaire de spécifier l'origine du moment que ce style aient été inclus ici. Ce type d'information sera documenté dans la versioning public API du composant.